Detectors/ITSMFT/ITS/workflow/src/DCSParserSpec.cxx

```diff
@@ -470,19 +470,15 @@ void ITSDCSParser::pushToCCDB(ProcessingContext& pc)
   // Timestamps for CCDB entry
   long tstart = 0, tend = 0;
   // retireve run start/stop times from CCDB
-  o2::ccdb::CcdbApi api;
-  api.init(mCcdbUrlRct);
-  // Initialize empty metadata object for search
-  std::map<std::string, std::string> metadata;
-  std::map<std::string, std::string> headers = api.retrieveHeaders(
-    "RCT/Info/RunInformation", metadata, this->mRunNumber);
-  if (headers.empty()) { // No CCDB entry is found
-    LOG(error) << "Failed to retrieve headers from CCDB with run number " << this->mRunNumber
-               << "\nWill default to using the current time for timestamp information";
+  auto& cdbman = o2::ccdb::BasicCCDBManager::instance();
+  cdbman.setURL(mCcdbUrlRct);
```

Comment:
@chiarazampolli No, I also set the correct ccdb server.

Reply:
But it was the same before, just via the CcdbApi.

Reply:
It was failing on the BasicCcdbManager::getRunDuration, with the manager set to the production host.

That said, I've noticed that the getRunDuration does not use setFatalWhenNull setting of the manager but has its own argument for that. So, if the RCT entry is missing, it will still fail.

Reply:
Ah ok, from the diff I did not see that the manager was set to another URL.

For the second comment, it might than be that this PR is not sufficient, but typically it should. I guess @iravasen will change it in another PR?

Reply:
The production server is default... For avoiding fatal on calibration runs: https://github.com/AliceO2Group/AliceO2/pull/13230